Scarb.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "gmx_referral_cairo"
```

Comment:
remove 'gmx' mentions from package name.

Reply:
changed it to jediswap_referral